libs/notifications/pom.xml

```diff
@@ -93,7 +95,7 @@
         <dependency>
```

Comment:
Any reason we're adding `mockito-core` as a dependency here? I'd expect `mockito-all` to make this unnecessary.